applications/box.md

```diff
@@ -0,0 +1,213 @@
+
+![https://dev.to/fx/google-photos-open-source-alternative-with-react-native-80c](https://raw.githubusercontent.com/functionland/blog/main/photos-intro-dev-to/photos.gif)
+
+* Data models / API specifications of the core functionality
+
+![https://dev.to/fx/google-photos-open-source-alternative-with-react-native-80c](https://raw.githubusercontent.com/functionland/functionland.github.io/main/static/images/box-diagram.jpg)
+
+* An overview of the technology stack to be used
+
+"box" is a collection of integrations for creating a server software that runs js-ipfs in a Node.js environment to serve application data. We use libp2p to create the required protocols for front-end developers to interface with the back-end "box" node, which is running on ipfs. Developers can send commands such as fetch files, upload, or access database. This is achieved by importing a single library that exposes the "box" functionalities to them, @functionland/graph (inspired by [Microsoft Graph workflow](https://docs.microsoft.com/en-us/graph/use-the-api)). These commands are then transferred to "box" node using libp2p. On the back-end the commands are interpreted by the "box" and are sent to js-ipfs or database library to get executed. Database library is using IPLD, specifically DAG-JSON/DAG-JOSE. Multiple "box"es are synchronized and accessible using ipfs-cluster.
+
+Finally, as an availability guarantee for sensitive data, "box" provides the capability to pin data with Crust network, to give them a peace of mind over data safety. They have the option to backup their data to Filecoin network to preserve their important information from data loss.
+
+
+* Documentation of core components, protocols, architecture, etc. to be deployed
+* PoC/MVP or other relevant prior work or research on the topic:
+
+You can find the initial code for box here: [https://github.com/functionland/box](https://github.com/functionland/box)
+
+### Ecosystem Fit 
+
+Help us locate your project in the Crust landscape and what problems it tries to solve by answering each of these questions:
+
+* Where and how does your project fit into the ecosystem? We want to make it easier for front-end developers to build appications for decentralized web as well as close the gap between non-tech-savvy individuals and decentralized web. For this purpose we combine IPFS, Libp2p, IPLD and Crust to build the platform.
+* What need(s) does your project meet? 
+* Are there any other projects similar to yours in the IPFS/Crust ecosystem? No, closest ones are FleekHQ and Ceramic.
+  * If so, how is your project different? They provide a wrapper that connects developer to a certain backend (like IPFS for file storage and gunDb for database) at an extra cost to both end-users and developer. We are developing a protocol and not just a wrapper and give the option to user to set up this backend himself at no cost as a package.
```

Comment:
Could give more details about the comparison with Fleek and Textile?

Reply:
Hi, below is the answer, which I suggested as a commit
```suggestion
  * If so, how is your project different? Below is a comparison between "box" and simillar projects:
  
- **Fleek:** Managed IPFS hosting + gateway
- **Ceramic:** protocols and chain for data persistence + decentralized identity
- **Textile:** backend stack for building on IPFS and Filecoin + backend-as-a-service
- **Box:** on-premise Web 3 with protocols for a Client-Server model + integrations for hybrid on-premise and on-chain options
```